### ISSUE TITLE
Model with a column "size" will break the serializer as it thinks @resource is an array.

### ISSUE THREAD

**Issue by AlbaHoo** [jsonapi-serializer/jsonapi-serializer#104]
## Steps to Reproduce the Problem

Here is the situation,  I have a model called "DeviceSecondaryModel" and it has a column name is "size", this is causing issues,

I have to dig into the source code and found this:
```
    def is_collection?(resource, force_is_collection = nil)
      return force_is_collection unless force_is_collection.nil?

      resource.respond_to?(:size) && !resource.respond_to?(:each_pair)
    end

```
So basically it treated my single resource as array, as it has a size method.  

I don't know this is a bug report or a feature request, just want to highlight here in case others have the same issue.

## Specifications

  - Version:
  - Ruby version:

**Comment by jkogara** [comment]
I think this can be closed - there was a fix for this in 0edebd99 - @AlbaHoo looks like you're running an older version?

**Comment by stas** [comment]
We switched the collection detection to always test for an `Enumerable` from now on. 

Closing this as this should be more than enough to cover the use-cases.
